### DISCUSSION TITLE
Is there any way to customize theme?

### DISCUSSION THREAD

**Discussion by QuAzI** [wallabag/wallabag#7683] [Q&A]
I want to customize some elements
![image](https://github.com/user-attachments/assets/1687e4b5-8545-4274-8ba2-ef13b7d87419)

**Comment by j0k3r** [comment]
https://doc.wallabag.org/en/admin/custom_css.html :)

**Reply by QuAzI** [reply]
So I can't use it on app.wallabag.it, I have to install my own self-hosted instance?

**Reply by j0k3r** [reply]
Yes that's right.
Otherwise you can use script on the browser level to affect only your use case, like https://greasyfork.org/
